my-app/src/constants/Game.jsx

```diff
@@ -0,0 +1,21 @@
+export const urlSendData = 'http://localhost:8080/api/aiProcessor';
+export const urlSendRole = 'http://localhost:8080/api/aiRole';
+
+export const SQUARE_CLICKED = 'SQUARE_CLICKED';
+export const BEFORE_SEE_HISTORY = 'BEFORE_SEE_HISTORY';
```

Comment:
let's rename some actions (to give them more meaningful usage):
please rename `BEFORE_SEE_HISTORY` to `JUMP_TO_HISTORY_STEP`